guides/doc-Configuring_User_Authentication/docinfo.xml

```diff
@@ -0,0 +1,15 @@
+<title>{ConfiguringUserAuthenticationDocTitle}</title>
+<productname>Red Hat Satellite</productname>
+<productnumber>{ProjectVersion}</productnumber>
+<subtitle>Configure authentication for {Project} users and enable authentication features such as single sign-on or two-factor authentication.</subtitle>
```

Comment:
I recommend a shorter subtitle: "Configure authentication for {Project} users and enable authentication features". 

IMO, not necessary to provide examples because authentication is a well-known subject.

Reply:
I know what you mean, except that we do have requests to be more explicit about the authentication features available for users (https://github.com/theforeman/foreman-documentation/pull/3222 was also related to that). So I'd prefer to keep them to make it easier for users to look up important keywords.